chapters/publication.tex

```diff
@@ -1,96 +1,178 @@
+Holding code and data to the same standards as written work
+is a new discipline for many researchers,
+and here we provide some basic guidelines and responsibilities for that process
+that will help you prepare a functioning and informative replication package.
+In all cases, we note that technology is rapidly evolving
+and that the specific tools noted here may not remain cutting-edge,
+but the core principles involved in publication and transparency will endure.
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Collaborating on academic writing}
+\section{Collaborating on technical writing}
+
+It is increasingly rare that a single author will prepare an entire manuscript alone.
+More often than not, documents will pass back and forth between several writers
+before they are ready for publication,
+so it is essential to use technology and workflows that avoid conflicts.
+Just as with the preparation of analytical outputs,
+this means adopting tools and practices that enable tasks
+such as version control and simultaneous contribution.
+Furthermore, it means preparing documents that are \textbf{dynamic} --
+meaning that updates to the analytical outputs that constitute them
+can be passed on to the final output with a single process,
+rather than copy-and-pasted or otherwise handled individually.
+Thinking of the writing process in this way
+is intended to improve organization and reduce error,
+such that there is no risk of materials being compiled
+with out-of-date results, or of completed work being lost or redundant.
+
+\subsection{Dynamic documents}
+
+Dynamic documents are a broad class of tools that enable a streamlined, reproducible workflow.
+The term ``dynamic'' can refer to any document-creation technology
+that allows the inclusion of explicitly encoded linkages to raw output files.
+This means that, whenever outputs are updated,
+the next time the document is loaded or compiled, it will automatically include
+all changes made to all outputs without any additional intervention from the user.
+This means that updates will never be accidentally excluded,
+and it further means that updating results will not become more difficult
+as the number of inputs grows,
+because they are all managed by a single integrated process.
+
+You will note that this is not possible in tools like Microsoft Office,
+although there are various tools and add-ons that produce similar functionality,
+and we will introduce some later in this book.
+In Word, by default, you have to copy and paste each object individually
+whenever tables, graphs, or other inputs have to be updated.
+This creates complex inefficiency: updates may be accidentally excluded
+and ensuring they are not will become more difficult as the document grows.
+As time goes on, it therefore becomes more and more likely
+that a mistake will be made or something will be missed.
+Furthermore, it is very hard to simultaneously edit or track changes
```

Comment:
well OneDrive does allow for easy simultaneous edits

Reply:
no